UM/Signal.py

```diff
@@ -542,7 +542,7 @@ def append(self, left_item: T, right_item: U) -> "WeakImmutablePairList[T,U]":
         """
 
         new_instance = WeakImmutablePairList()  # type: WeakImmutablePairList[T,U]
-        new_instance.__list = self.__cleanList()
+        new_instance.__list = self.__list.copy()
```

Comment:
This also happens in `WeakImmutableList` (as opposed to `WeakImmutablePairList`). I'll put the same change there then, with the same reasoning. (Unless that messes up my tests.)